resources/lang/en/patrols/general/border.json

```
@@ -219,7 +219,7 @@
         "chance_of_success": 50,
         "success_outcomes": [
             {
-                "text": "Upon closer inspection, it was just remarking from a clannmate.",
+                "text": "Upon closer inspection, it was just remarking from a clanmate.",
```

```suggestion
                "text": "Upon closer inspection, it was just remarking from a Clanmate.",
```
Clan and Clanmate are always capitalized, just fyi!